.azure/pipelines/examples-test.yml

```diff
@@ -130,6 +130,51 @@ jobs:
           ./testdata/multi_terraform_workspace_config_file_comment_golden.md
           /tmp/infracost_comment.md
         displayName: Check the comment
+  - job: open_policy_agent
+    displayName: Open Policy Agent
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+      - task: InfracostSetup@0
+        displayName: Setup Infracost
+        inputs:
+          apiKey: $(apiKey)
+      - bash: >
+          INSTALL_LOCATION="$(Build.SourcesDirectory)/bin"
+
+          mkdir -p ${INSTALL_LOCATION}
+
+
+          wget -O ${INSTALL_LOCATION}/opa
+          https://openpolicyagent.org/downloads/v${OPA_VERSION}/opa_linux_amd64_static 
+
+          chmod +x ${INSTALL_LOCATION}/opa
+
+
+          echo "##vso[task.setvariable
+          variable=PATH;]$(PATH):${INSTALL_LOCATION}"
+        displayName: Setup OPA
+        env:
+          OPA_VERSION: 0.35.0
+      - bash: >-
+          infracost breakdown --path=examples/opa/code/plan.json --format=json
+          --out-file=/tmp/infracost.json
+        displayName: Run Infracost
+      - bash: >-
+          opa eval --input /tmp/infracost.json -d
+          examples/opa/policy/policy.rego --format pretty "data.infracost.deny"
+          | tee /tmp/opa.out
+        displayName: Run OPA
+      - bash: |
+          denyReasons=$(</tmp/opa.out)
+          if [ "$denyReasons" != "[]" ]; then
+            echo "##vso[task.logissue type=error]Policy check failed"
+            echo "##vso[task.logissue type=error]$denyReasons"
```

Comment:
The output shows this, do we need to iterate through the list? https://dev.azure.com/infracost/base/_build/results?buildId=550&view=logs&j=d0ee13d9-3503-51dd-3607-d7bccb219083&t=a4365896-62ee-59cd-f8d1-dcde0547daba&l=13

> ##[error]n

Reply:
That output was just for testing, the actual output would be something like `"['resource_1', 'resource_2']"`